core/l10n.js

```diff
@@ -59,8 +59,10 @@ define('core/l10n',
 
     return {
         getDirection: function(context) {
+            // Try to get browser language, if undefined use default locale
+            var browserLanguage = window.navigator.language || 'en-US';
             var language = context ? context.language :
-                                     window.navigator.l10n.language;
+                                     browserLanguage;
```

Comment:
l10n_init.js should never let window.navigator.l10n become undefined, so if that's an issue, let's make sure we address it there.